pkg/oauthclientscontroller/oauthclientscontroller.go

```diff
@@ -0,0 +1,147 @@
+package oauthclientscontroller
+
+import (
+	"monis.app/go/openshift/controller"
```

Comment:
wtf...

Reply:
yeah, I did not notice that move in our operators originally, either :man_shrugging: